club-website/src/app/page.tsx

```diff
@@ -1,24 +1,91 @@
+'use client';
 import Image from "next/image";
+import { poppins } from "./fonts"
+import { useState, useRef, useEffect } from 'react'
+import { FaYoutube, FaGoogleDrive, FaGithub } from "react-icons/fa";
 
-export default function Resources() {
-  return (
-    <main className="bg-slate-50 flex min-h-screen flex-col space-y-1 gap-12">
+interface MaterialHub {
+  title: string;
+  icon: React.ReactElement;
+  image: React.ReactElement;
+  link: string;
+  description: string;
+}
+
+const WINDOW_SIZE_min = "min-w-[300px] md:min-w-[400px] lg:min-w-[600px] "
+const WINDOW_SIZE_max = "max-w-[300px] md:max-w-[400px] lg:max-w-[600px] "
+
+const MATERIAL_HUBS : MaterialHub[] = 
+[
+  {
+    title: 'Our YouTube',
+    icon: <FaYoutube size={24} />,
+    image: <Image src='/yt_vid.png' alt="Youtube video" width={500} height={200} className="w-full"></Image>,
+    link:"https://www.youtube.com/@engrtao",
+    description: "TAO’s YouTube channel (TAO-ENGR) -- including basic videos as well as recordings of exam reviews.",
+  },
+  {
+    title: 'Our Google Drive',
+    icon: <FaGoogleDrive size={24} />,
+    image: <Image src='/drive_ss.png' alt="Google drive" width={500} height={200} className="w-full"></Image>,
+    link: "https://drive.google.com/drive/folders/128TePaIK-zwts9vzNhZfj3_mJ3IH_L9d?usp=sharing",
+    description: "Access TAO’s shared resources including review materials and practice problems on tx.ag/taodrive.",
+  },
+  {
```

Comment:
![image](https://github.com/TAO-ENGR/club-website/assets/91210043/7dbdc7ac-c1a1-43e6-b17f-df4bfded6555)

switch out apostrophes to `&apos;` to make eslint happy